client/post-editor/editor-ground-control/index.jsx

```diff
@@ -26,7 +26,7 @@ const Card = require( 'components/card' ),
 import { setDate } from 'state/ui/editor/post/actions';
 
 function isPostEmpty( props ) {
-	return ( props.isNew && ! props.isDirty ) || ! props.hasContent;
+	return ! props.hasContent || ( ! props.isNew && ! props.isDirty );
```

Comment:
Yeah, this is / was used to determine when to light up the publish and preview buttons. The logic is there is pretty convoluted, but covered by tests, and I didn't want to mess with it too much.